src/h5web/visualizations/heatmap/ColorBar.tsx

```diff
@@ -28,12 +33,10 @@ function ColorBar(props: Props): JSX.Element {
       />
       <svg className={styles.colorBarAxis} height={gradientHeight} width="2em">
         <AxisRight
-          scale={scaleLinear()
-            .domain(dataBounds)
-            .range([gradientHeight, 0])
-            .nice()}
-          strokeWidth={0}
+          scale={scale}
+          hideAxisLine
           numTicks={adaptedNumTicks(gradientHeight)}
+          tickFormat={scale.tickFormat(adaptedNumTicks(gradientHeight), '.3')}
```

Comment:
Ticks are a bit in funny places with the symlog scale. I think this would need some dedication to get something nice.